src/Servers/Kestrel/samples/QuicSampleClient/Program.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Buffers;
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Connections.Features;
+using Microsoft.AspNetCore.Server.Kestrel.Https;
+using Microsoft.AspNetCore.Server.Kestrel.Transport.MsQuic;
+
+namespace QuicSampleClient
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var transportOptions = new MsQuicTransportOptions()
+            {
+                Alpn = "QuicTest",
+                RegistrationName = "Quic-AspNetCore-client",
+                Certificate = CertificateLoader.LoadFromStoreCert("localhost", StoreName.My.ToString(), StoreLocation.CurrentUser, true),
```

Comment:
Are you going to check in a cert?

Reply:
This is the dev cert usually.

Reply:
The dev cert is OK for samples, just not tests.